packages/config/src/balance/balance.ts

```diff
@@ -80,19 +84,47 @@ function system(): SystemBalanceConfig {
   };
 }
 
-function tokens<Symbols extends AssetSymbol = AssetSymbol>(
-  asset: number | bigint | Symbols | 'MOVR' | 'KUSD' | 'AUSD',
+function tokens<Symbols extends AssetSymbol = AssetSymbol>() {
+  return {
+    token: (asset: Symbols | 'MOVR' | 'KUSD' | 'AUSD') => token<Symbols>(asset),
+    foreignAsset: (asset: AssetId | Symbols) => foreignAsset<Symbols>(asset),
+    miningResource: (asset: AssetId) => miningResource<Symbols>(asset),
+    fungibleToken: (asset: AssetId) => fungibleToken<Symbols>(asset),
+  };
+}
+
+function token<Symbols extends AssetSymbol = AssetSymbol>(
+  asset: Symbols | 'MOVR' | 'KUSD' | 'AUSD',
+) {
+  return tokensBase<Symbols>({ [BalanceCurrencyTypes.Token]: asset });
+}
+
+function foreignAsset<Symbols extends AssetSymbol = AssetSymbol>(
+  asset: AssetId,
+) {
+  return tokensBase<Symbols>({ [BalanceCurrencyTypes.ForeignAsset]: asset });
+}
+
+function miningResource<Symbols extends AssetSymbol = AssetSymbol>(
+  asset: AssetId,
+) {
+  return tokensBase<Symbols>({ [BalanceCurrencyTypes.MiningResource]: asset });
+}
+
+function fungibleToken<Symbols extends AssetSymbol = AssetSymbol>(
+  asset: AssetId,
+) {
+  return tokensBase<Symbols>({ [BalanceCurrencyTypes.FungibleToken]: asset });
+}
```

Comment:
I think is shorter just call directly `tokensBase`. 
```suggestion
function tokens<Symbols extends AssetSymbol = AssetSymbol>() {
  return {
    token: (asset: Symbols | 'MOVR' | 'KUSD' | 'AUSD') =>
      tokensBase<Symbols>({ [BalanceCurrencyTypes.Token]: asset }),
    foreignAsset: (asset: AssetId | Symbols) =>
      tokensBase<Symbols>({ [BalanceCurrencyTypes.ForeignAsset]: asset }),
    miningResource: (asset: AssetId) =>
      tokensBase<Symbols>({ [BalanceCurrencyTypes.MiningResource]: asset }),
    fungibleToken: (asset: AssetId) =>
      tokensBase<Symbols>({ [BalanceCurrencyTypes.FungibleToken]: asset }),
  };
}
```

Reply:
oh yeah, good point